### PR TITLE
Feature/initial release

### PR DESCRIPTION
**Pull request by atxplangl**

### CODE REVIEW

File: examples/complete/main.tf [hunk 1]
```diff
@@ -1,23 +1,67 @@
-data "aws_vpcs" "this" {
-  tags = {
-    purpose = "vega"
+provider "aws" {
```
Comment (reeves122) [inline examples/complete/main.tf]: This provider block shouldn't be necessary

File: variables.tf [hunk 2]
```diff
@@ -1,9 +1,72 @@
...
+  default     = "rate(1 minute)"
 }
 
-variable "vpc_id" {
-  description = "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group#vpc_id"
+variable "schedule_expression_timezone" {
+  description = "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule#schedule_expression"
+  type        = string
+  default     = "America/New_York"
+
```
Comment (reeves122) [inline variables.tf]: extra space?

File: variables.tf [hunk 3]
```diff
@@ -1,9 +1,72 @@
...
+  type        = string
+  default     = "rate(1 minute)"
 }
 
-variable "vpc_id" {
-  description = "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group#vpc_id"
+variable "schedule_expression_timezone" {
+  description = "https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule#schedule_expression"
+  type        = string
+  default     = "America/New_York"
```
Comment (reeves122) [inline variables.tf]: Perhaps the default should be UTC?

**Review by reeves122 — CHANGES_REQUESTED** [review]

File: examples/complete/main.tf (hunk 1 above)
Comment (plangley-champ) [reply examples/complete/main.tf]: done

File: variables.tf (hunk 2 above)
Comment (plangley-champ) [reply variables.tf]: removed

File: variables.tf (hunk 3 above)
Comment (plangley-champ) [reply variables.tf]: setting to null which is UTC

**Review by reeves122 — APPROVED** [review]

**Pull request merged**